.travis.yml

```diff
@@ -18,6 +18,7 @@ env:
   - EMBER_TRY_SCENARIO=ember-release
   - EMBER_TRY_SCENARIO=ember-beta
   - EMBER_TRY_SCENARIO=ember-canary
+  - EMBER_TRY_SCENARIO=ember-default
```

Comment:
I'm not sure if that's necessary for `ember-simple-auth`. Do we use any private APIs in this project that we need to make sure are available in all minor releases? In the other projects it makes sense since we use Handlebars AST transforms which are not public API yet.